packages/polyfill/package-lock.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Pretty sure this shouldn't be committed right? None of the other packages have a lock file